site/content/en/docs/user/stages-configuration.md

```diff
@@ -0,0 +1,105 @@
+# Stages Configuration
+
+{{< hint "info" >}}
+
+This document walks you through how to configure the Stages of Lifecycle
+
+{{< /hint >}}
+
+## What is a Stage?
+
+The Stage API is a [Kwok Configuration](./configuration) that allows users to define and simulate different stages in the lifecycle of Kubernetes resources, such as nodes and pods.
+Each Stage resource specifies a resourceRef field that identifies the type of resource that the stage applies to, and a selector field that determines when the stage should be executed.
+
+A Stage resource has the following fields:
+
+``` yaml
+kind: Stage
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: <string>
+spec:
+  resourceRef:
+    apiGroup: <string>
+    kind: <string>
+  selector:
+    matchLabels:
+      <string>: <string>
+    matchAnnotations:
+      <string>: <string>
+    matchExpressions:
+      - key: <expressions-string>
+        operator: <string>
+        values:
+          - <string>
+  delay:
+    durationSeconds: <int>
+    durationSecondsFrom: <expressions-string>
+    jitterDurationSeconds: <int>
+    jitterDurationSecondsFrom: <expressions-string>
```

Comment:
```suggestion
  delay:
    duration: <duration-string>
    durationFrom: <expressions-string>
    jitterDuration: <duration-string>
    jitterDurationFrom: <expressions-string>
```

Reply:
The old format was right - [api-conventions#units](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units)